packages/maskbook/src/social-network/types.ts

```diff
@@ -103,6 +103,8 @@ export namespace SocialNetworkUI {
             postInspector?(signal: AbortSignal, current: PostInfo): void
             /** Inject a toolbar that displayed on the top of the page */
             toolbar?(signal: AbortSignal): void
+            /** Inject a tool box that displayed in the twitter navigation bar */
```

Comment:
```suggestion
            /** Inject a tool box that displayed in the navigation bar of the SNS */
```

This is not only for Twitter.